plugins/registry/combine.go

```diff
@@ -0,0 +1,55 @@
+package registry
+
+import (
+	"github.com/laszlocph/woodpecker/model"
+)
+
+type combined struct {
+	registries []model.ReadOnlyRegistryService
+	dbRegistry model.RegistryService
+}
+
+func Combined(dbRegistry model.RegistryService, registries ...model.ReadOnlyRegistryService) model.RegistryService {
+	registries = append(registries, dbRegistry)
+	return &combined{
+		registries: registries,
+		dbRegistry: dbRegistry,
+	}
+}
+
+func (c combined) RegistryFind(repo *model.Repo, name string) (*model.Registry, error) {
+	for _, registry := range c.registries {
```

Comment:
Looks as if only the filesystem registries are looped through. The call should be delegated to `dbRegistry` too, shouldn't it?

Reply:
Shouldn't have to, dbRegistry is added to the list https://github.com/laszlocph/woodpecker/pull/120/files#diff-60ca29804ed147e90820993d6f198f1fR13

Reply:
My bad. Thanks for clarification.